app/views/generic_files/_media_display.html.erb

```diff
@@ -0,0 +1,33 @@
+<% if @generic_file.image? %>
+
+  <%= render_download_icon "Download the full-sized image" %>
+
+<% elsif @generic_file.video? %>
+
+  <video controls="controls" class="video-js vjs-default-skin" data-setup="{}" preload="auto">
+    <source src="<%= download_path(@generic_file, file: 'webm') %>" type="video/webm" />
+    <source src="<%= download_path(@generic_file, file: 'mp4') %>" type="video/mp4" />
+    Your browser does not support the video tag.
```

Comment:
We need to override this partial, because Sufia's version is hardcoded to call sufia.download_path, which breaks streaming with the new download routes